proto/hummock.proto

```diff
@@ -706,6 +699,12 @@ message RiseCtlListCompactTaskAssignmentResponse {
   repeated CompactTaskAssignment task_assignment = 1;
 }
 
+message RiseCtlListCompactTaskProgressRequest {}
```

Comment:
nits: it is better not calling it `RiseCtl...Requqest/Response` if it is not used in risectl